mkdocs/docs/HPC/conda.md

```diff
@@ -0,0 +1,58 @@
+# Conda
+
+Conda is a software package manager for data science that allows unprivileged (non-administrative) users to search, 
+fetch, install, upgrade, use, and manage supported open-source software packages and 
+programming languages/libraries/environments in a directory they have write access to. 
+Conda allows users to create reproducible scientific software environments
+
+We do not recommend using conda environments on HPC clusters due to several common issues, 
+which we will explain in this section. However, we do not forbid the use of conda entirely. In certain situations, 
```

Comment:
There are no restrictions on the usage of conda, everyone can do so if they want.

Reply:
solved in [cdd603f](https://github.com/hpcugent/vsc_user_docs/pull/649/commits/cdd603f1240d04ed1b18b28963c5f4dab49b9bf5)